### DISCUSSION TITLE
How to configure proxy?

### DISCUSSION THREAD

**Discussion by fung** [apideck-libraries/portman#453] [Q&A] [answered]
Hi,

I am currently using this tool behind corporate proxy. May I know how to set proxy for the cli?

Thanks!

**Comment by thim81** [comment, marked as answer]
hi @fung 

Portman generates a Postman collection, that can be imported (manually or automatic) in Postman or executed via the Newman CLI.
To configure the usage of a proxy, we would advise to review the Postman documentation on how to leverage a proxy for making requests: https://learning.postman.com/docs/getting-started/proxy/

**Reply by fung** [reply]
hi @thim81 

Thank you for your reply!
